.ci/ci.yml.tmpl

```diff
@@ -2,6 +2,13 @@
 
 # These resource types are here until the PRs get merged in upstream.  :)
 resource_types:
+    - name: merged-downstreams
+      type: docker-image
+      source:
+          # Note: resource types cannot use credhub substitution - "nmckinley" is hardcoded here.
```

Comment:
:(

Reply:
Sorry!  Someone else can figure out how to get these things uploaded to GCR and then it won't be so sad, but I'm not planning on it in the near future.